explore-reports-and-create-documentation/audit-reports/audit-dashboard.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Audit Dashboard
+description: This article describes Audit Dashboard and its features.
+author: Janko Ban
+date: 14/12/2018
+---
+Audit Dashboard is giving you an overview of the audit actions in the farm for the last 7 days, drillable to specific detailed reports.
+
+Audit Dashboard makes monitoring of the audit actions easier and more intuitive.
+
+The following tiles with audit actions are visible:
+
+* Site Collections with Audit enabled/disabled
+* Number of SharePoint groups created
+* Number of broken Permission Inheritance's
+* Top Users by Activity
+* Top Site Collections by Changes
+* Number of Document Views
+* Number of Changes per Event type
+* Top 5 Administrative actions performed on the farm
+
+__Site Collections with Audit enabled/disabled__ tile shows you a number of Site Collections with Audit options enabled/disabled. Clicking on the tile drills to the Audit Settings report.
+
+Tile is disabled when all Site Collections audit settings are disabled.
+
+__SP Groups Created and Number of Permissions Inheritance Broken__ tile shows a number of Sharepoint Groups Created and Permission Inheritance Broken. Clicking on the tile drills to the Audit Log Details report.
```

Comment:
budući da više tileova drilla na audit log details report, možda napisati i da će se filter postaviti 
npr. 
...to the Audit Log Details report with the Group Created option preselected in the Event Type filter.

Slično tome napraviti i na ostalim mjestima gdje se drilla na audit log report..